_episodes/017-twir-341-342.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "This Week in Rust - Issue 341 and 342"
+date: 2020-06-09T14:30:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/twir-2020-05-26.mp3
+duration: "11:21"
+length: "7780645"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Nell Shamrell-Harrington — lead editor of This Week in Rust — takes you through highlights from this week's issue of TWiR.
+
+<!--
+The episode introduction goes here.
+The first paragraph should ideally be short, and is used in various
+places as a "short description" for the episode. Any subsequent
+paragraphs show up as "expanded description".
+-->
+
+### Contributing to Rustacean Station
+
+<!-- You can probably leave this as-is -->
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Referenced resources
+
+#### Issue 341
+
+- [This Week in Rust 341](https://this-week-in-rust.org/blog/2020/06/02/this-week-in-rust-341/)
+- [RustConf](https://rustconf.com/)
+- [Rust Contributor Survey](https://docs.google.com/forms/d/e/1FAIpQLSek3vddlYf3YFbV1RVavm9qAJhGxw-Zi_XJ8RBIhNUuuf34Pw/viewform)
+- [A Retrospective on the 2018 rust-lang.org redesign](https://blog.rust-lang.org/inside-rust/2020/05/26/website-retrospective.html)
+- [Contributing to Rust](https://blog.elinvynia.com/posts/2020-05-26-contributing-to-rust.html)
+- [How to build a WebSocket server with Rust](https://blog.logrocket.com/how-to-build-a-websocket-server-with-rust/)
+- [Custom Types in Diesel](https://kitsu.me/posts/2020_05_24_custom_types_in_diesel)
+- [Fuzzing Sequoia-PGP](https://blog.hackeriet.no/fuzzing-sequoia/)
+- [Sorting algorithms in Rust](https://dev.to/jlkiri/sorting-algorithms-in-rust-1386)
+- [3D boids swimming in perfect harmony: Implementing the boids flocking algorithm in Rust](https://www.reddit.com/r/rust/comments/gsldbi/3d_boids_swimming_along_in_perfect_harmony/)
+- [Aprende Rust en español](https://dev.to/robertohuertasm/aprende-rust-en-espanol-1pea)
+- [A Rust and WASM tutorial on building Bitcoin infrastructure](https://www.youtube.com/watch?v=qaykNPHJcyw)
+- [Crust of Rust: Iterators](https://www.youtube.com/watch?v=yozQ9C69pNs&feature=emb_logo)
+- [Rust and Tell Berlin - May 2020](https://www.youtube.com/watch?v=rpilJV-eIVw&feature=emb_logo)
+
+#### Issue 342
```

Comment:
Nice! Just for future reference, you can also include timestamps here of the form `[@15:09]`, and they will be converted into clickable links that jump in the player on the website. No big need for that in such short episodes, but something to keep in mind :)